frontend/src/components/AdminPane/index.js

```
@@ -0,0 +1,39 @@
+import {connect} from "react-redux";
+import {componentsOperations} from "../duck";
+import {Component} from "react";
+import adminPane from "./presenter.jsx"
+
+const mapStateToProps = (state) => {
+    const {
```

ESLint

[error] Expected indentation of 2 spaces but found 4.

(indent)